src/main/java/org/jboss/windup/plugin/WindupMojo.java

```diff
@@ -73,7 +79,7 @@
     /**
      * Location of the input file application.
      */
-    @Parameter(alias = "input", property = "input", required = true)
+    @Parameter( defaultValue = "${project.build.sourceDirectory}", property = "input", required = true )
```

Comment:
+1
even I prefer to have the very same maven plugin configuration options to match CLI options names to not confuse users and to not require the additional documentation for mapping CLI/Maven plugin config.